app/controllers/tests_controller.rb

```diff
@@ -0,0 +1,49 @@
+class TestsController < ApplicationController
+  before_action :find_test, only: %i[show destroy edit update]
+
+  def index
+    @tests = Test.all
+  end
+
+  def new
+    @test = Test.new
+  end
+
+  def create
+    @test = Test.new(test_params)
+    @test.user_id = 1
```

Comment:
не надо хардкодить ай-ди, можно "не угадать"
пользуйся объектами из базы
например здесь можно так:
```suggestion
    @test.user = User.first
```
а еще лучше написать отдельный метод, назвать его как-то типа `user`, `current_user` и здесь передать через ассоциацию авторство:
```ruby
    @test = current_user.tests.new(test_params)
```
такая работа через ассоциации в рельсах стандартный стиль